api/input/isrbxactive.md

```diff
@@ -1,7 +1,7 @@
 # isrbxactive
 **Signature:** `isrbxactive(): boolean` <br>
 **Aliases:** isgameactive <br>
-**Description:** Returns true is the game is in focus (this must be true for other input functions to work) <br>
+**Description:** Returns true if ROBLOX's window is in focus (this must be true for other input functions to work) <br>
```

Comment:
Brand references are not allowed. Wording using "game" and/or related is intended.

Reply:
I modified the pull request to fix this.